src/main.rs

```diff
@@ -162,24 +166,93 @@ struct Args {
 
 impl Args {
 
+    pub fn defaults() -> Args {
+        Args {
+            verbose: false,
+            brightness: (DEFAULT_BRIGHTNESS, DEFAULT_BRIGHTNESS),
+            gamma: (DEFAULT_GAMMA, DEFAULT_GAMMA, DEFAULT_GAMMA),
+            location: location::Location::new(55.7, 12.6),
+            method: None,
+            temperatures: (DEFAULT_DAY_TEMP, DEFAULT_NIGHT_TEMP),
+            transition: true,
+            mode: Mode::Continual,
+        }
+    }
+
+    pub fn update_from_config(mut self) -> Result<Args> {
+        let conf = std::env::home_dir()
+            .map(|mut path| { path.push(".config/redshift.conf"); path })
+            .and_then(|home| ini::Ini::load_from_file(&home).ok());
+
+        if let Some(conf) = conf {
+            if let Some(brightness_day) = conf.get_from(Some("redshift"), "brightness-day") {
```

Comment:
`Some("redshift")` is used everywhere here. Is there a way in ini to extract a particular section?